src/contexts/auth-provider.js

```diff
@@ -11,6 +11,13 @@ const AuthProvider = ({ children }) => {
   const [loaded, setLoaded] = useState(false);
   const [trigger, setTrigger] = useState(false);
 
+  const resetUserData = () => {
+    setTrigger(false);
```

Comment:
I think `setTrigger(!trigger)` solves the forever loading problem right after logging out.

Reply:
I tried that, it didn't log out but kept redirecting me to the dashboard. It doesn't seem to empty the values.

Reply:
This bug was in the file `auth-provider`, I removed `setLoaded()` from reset values and it got resolved.

Reply:
Interesting. It was working for me... We can take a look again tomorrow

Reply:
@harmanwho, sorry. I had forgotten to check this again but it looks like it's working well with the last change you made 👍